pallets/asset/src/lib.rs

```diff
@@ -356,8 +357,8 @@ decl_storage! {
         /// The total asset ticker balance per identity.
         /// (ticker, DID) -> Balance
         pub BalanceOf get(fn balance_of): double_map hasher(blake2_128_concat) Ticker, hasher(blake2_128_concat) IdentityId => T::Balance;
-        /// A map of pairs of a ticker name and an `IdentifierType` to asset identifiers.
-        pub Identifiers get(fn identifiers): map hasher(blake2_128_concat) (Ticker, IdentifierType) => AssetIdentifier;
+        /// A map of a ticker name and asset identifiers.
+        pub Identifiers get(fn identifiers): map hasher(blake2_128_concat) Ticker => Vec<Identifier>;
```

Comment:
N.B. the storage modification here. -- should we add migration?

Reply:
@maxsam4 said that the data in the existing map is garbage so no need, feel free to correct me.

Reply:
@CJP10 Lets dump the old data though.

Something like 
`<Identifiers>::remove_all();`